crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -0,0 +1,792 @@
+# Nushell Config File
+#
+# version = "0.99.2"
```

Comment:
wrong version?

Reply:
Good catch. @NotTheDr01ds I'm also thinking all these config/env/scaffold/sample files should have a version at the top. Also, I'm getting an error in one. It should be `=` not `:`.
![image](https://github.com/user-attachments/assets/db0c2584-336d-4d74-a183-68967dddc96d)